lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -232,6 +232,13 @@ sub scenario_hash {
     return \%scenario;
 }
 
+sub scenario {
+    my ($self) = @_;
+    my $scenario = join('-', map { $self->get_column($_) } SCENARIO_KEYS);
+    $scenario .= "@" . $self->MACHINE if $self->MACHINE;
```

Comment:
Nitpick, but in Perl method calls are some of the most expensive operations, so it is best to avoid unnecessary ones.
```
if (my $machine = $self->MACHINE) { $scenario .= "@" . $machine }
```